packages/framework/src/Model/AdvancedSearchOrder/Filter/OrderLastNameFilter.php

```diff
@@ -70,11 +70,10 @@ public function extendQueryBuilder(QueryBuilder $queryBuilder, $rulesData)
      */
     protected function getContainsDqlOperator($operator)
     {
-        switch ($operator) {
-            case self::OPERATOR_CONTAINS:
-                return 'LIKE';
-            case self::OPERATOR_NOT_CONTAINS:
-                return 'NOT LIKE';
+        if ($operator === self::OPERATOR_CONTAINS) {
```

Comment:
Just wondering, if I call `getContainsDqlOperator('banana')`, I get `NOT LIKE`. Is it something we want?
(same in other filter files)

Reply:
I think so, because before it we got nothing. When I think about it, maybe it would be better to throw an exception than returning firmly 'NOT LIKE', isn't it?

Reply:
I think exception would be better than unexpected behavior

Reply:
5bdb3d64ef9536a56b0fb2bc456c6e7a00b57732